sklearn/metrics/tests/test_classification.py

```diff
@@ -955,6 +987,80 @@ def test_multilabel_jaccard_similarity_score():
     assert_equal(jaccard_similarity_score(y1, np.zeros(y1.shape)), 0)
     assert_equal(jaccard_similarity_score(y2, np.zeros(y1.shape)), 0)
 
+    y_true = np.array([[0, 1, 1], [1, 0, 0]])
+    y_pred = np.array([[1, 1, 1], [1, 0, 1]])
+    # average='macro'
+    assert_almost_equal(jaccard_similarity_score(y_true, y_pred,
+                                                average='macro'), 2. / 3)
+    # average='micro'
+    assert_almost_equal(jaccard_similarity_score(y_true, y_pred,
+                                                average='micro'), 3. / 5)
+    # average='samples' (default)
+    assert_almost_equal(jaccard_similarity_score(y_true, y_pred), 7. / 12)
+    assert_almost_equal(jaccard_similarity_score(y_true, y_pred,
+                                sample_weight=np.array([0.1, 0.9])), 31. / 60)
+    assert_almost_equal(jaccard_similarity_score(y_true, y_pred,
+                                                 average='samples',
+                                                 labels=[0, 2]), 1. / 2)
+    assert_almost_equal(jaccard_similarity_score(y_true, y_pred,
+                                                 average='samples',
+                                                 labels=[1, 2]), 1. / 2)
+    assert_almost_equal(jaccard_similarity_score(y_true, y_pred,
+                                                 average='samples',
+                                                 sample_weight=[1, 2]), 5. / 9)
```

Comment:
Don't bother testing sample weight. The common tests will

Reply:
Done.